NetKAN/WaterTanks.netkan

```diff
@@ -0,0 +1,31 @@
+{
+    "spec_version" : "v1.4",
+	"name"         : "Ballast Water Tanks",
+	"identifier"   : "WaterTanks",
+	"abstract"     : "Add parts with in-flight mass changing. Designed for submarines, act good as payload weight",
+	"author"       : "Jenyaza01",
+    "$kref"        : "#/ckan/github/jenyaza01/BallastWaterTanks",
+	"$vref"        : "#/ckan/ksp-avc",
```

Comment:
There's no .version file in the download, so this is not giving netkan any version info. You'll need to specify it manually or add an AVC .version file to the download.